### ISSUE TITLE
1.8.9 Sky.property with no fade in times set does not render sky

### ISSUE THREAD

**Issue by LazyKB** [sp614x/optifine#6733]
## Description of Issue
Sky.property files which do not have any fade in times set are not rendering at all. However the docs state that they will always render if this is the case. I tried  both writing the parameters and leaving them empty and completely removing the parameters. When using normal times the sky did render.

## Steps to Reproduce
In any sky texture folder, create a sky.properties file which does not have any fade in times specified. When the pack is loaded in game you will not have a custom sky.

## OptiFine Version
preview_OptiFine_1.8.9_HD_U_M6_pre2

## Installation Method
Forge Mod

## Fabric/Forge Version
11.15.1.2318

## Other Installed Mods
I verified this happens with no other mods

## F3 Debug Screenshot

![image](https://user-images.githubusercontent.com/87291643/161401725-f9ad3616-229f-4f17-985e-3159f835676e.png)

## Prior Testing
Custom skies do not work in Vanilla

## Additional Information
This is directly from the docs on Fade in times, just so you can read it here instead of going there if you like
# (Optional) Fade in/out times.  
# All times are in hh:mm 24-hour format, for reference:
#   Sunrise  =  6:00 = /time set 0
#   Noon     = 12:00 = /time set 6000
#   Sunset   = 18:00 = /time set 12000
#   Midnight =  0:00 = /time set 18000
# The fade times control the brightness when blending.
#   between startFadeIn and endFadeIn:   0 up to 1
#   between endFadeIn and startFadeOut:  always 1
#   between startFadeOut and endFadeOut: 1 down to 0
#   between endFadeOut and startFadeIn:  always 0
# Note that you do not need to specify startFadeOut; its value is uniquely
# determined by the other three.
# If no times are specified the layer is always rendered.
startFadeIn=<hh:mm>
endFadeIn=<hh:mm>
endFadeOut=<hh:mm>

**Comment by sp614x** [comment]
This was added for a recent 1.18.2 version, OptiFine for 1.8.9 doesn't support it yet.
The online documentation refers to the latest release (1.18.2).
The documentation inside the OptiFine JAR shows all features supported by this version.

**Comment by LazyKB** [comment]
That's all good. I was just documenting an issue. That's why I labeled it
as 1.8.9, I'm aware it's not maintained as much as latest version. I've
made a number of 1.8.9 bug reports. Just so you know it does have a
decently sized playerbase and updates to it would see use!

**Comment by fayer3** [comment]
with 1.8.9 the fade times are required, as stated in the doc
```
# (Required) Fade in/out times.  All times are in hh:mm 24-hour format.  For
# reference,
#   Sunrise  =  6:00 = /time set 0
#   Noon     = 12:00 = /time set 6000
#   Sunset   = 18:00 = /time set 12000
#   Midnight =  0:00 = /time set 18000
# The fade times control the brightness when blending.
#   between startFadeIn and endFadeIn:   0 up to 1
#   between endFadeIn and startFadeOut:  always 1
#   between startFadeOut and endFadeOut: 1 down to 0
#   between endFadeOut and startFadeIn:  always 0
# Note that you do not need to specify startFadeOut; its value is uniquely
# determined by the other three.
startFadeIn=<hh:mm>
endFadeIn=<hh:mm>
endFadeOut=<hh:mm>
```

**Comment by sp614x** [comment]
1.8.9 will be updated on the next round of updates and it will receive the updates that may be reasonably implemented in 1.8.x.
This fix will most probably be included.
